models/object_detection/requirements.txt

```diff
@@ -1,4 +1,5 @@
 -r ../../external/mmdetection/requirements/build.txt
```

Comment:
what about Polygon3?

Reply:
Polygon3 is missing on object detection dependencies. It isn't needed here.

Reply:
object detection, instance segmentation, text spotting is based on the same code base

Reply:
I think Polygon3 is not used in objct detection domain, since masks are not used.
Ilya, do you think that we need to add building Polygon3 here for uniformity?

Reply:
Polygon3 is not used anywhere except text_spotting, but it is exported everywhere 
https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/datasets/coco.py#L19 -> https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/core/evaluation/text_evaluation.py#L19

If you face with some problem in instance_segmentation case, then the same problem should be fixed in object_detection, should not it?

Is it possible that cache is not cleaned? that is probably why the problem might be hidden

Reply:
>> If you face with some problem in instance_segmentation case, then the same problem should be fixed in object_detection, should not it?
Ok
>> Is it possible that cache is not cleaned? that is probably why the problem might be hidden
No, only instance segmentation and text spotting had problems with Polygon3. Object detection was passed